CGPs/cgp-0024.md

```diff
@@ -7,6 +7,16 @@
 - Date Executed: [if executed]
 - Discussions to: https://forum.celo.org/t/ceur-celo-euro-goes-live/876
```

Comment:
Hey! Was going through all of the CGP updates, and they look great so far. I just noticed that this one looks like it has a few lines above the preamble and wanted to confirm - is that right?